internal/service/project/resource_project_test.go

```diff
@@ -491,22 +493,50 @@ func TestResourceProjectDependentsDeletingRefreshFunc(t *testing.T) {
 	}
 }
 
-const resourceName = "mongodbatlas_project.test"
+const (
+	resourceName         = "mongodbatlas_project.test"
+	dataSourceNameByID   = "data.mongodbatlas_project.test"
+	dataSourceNameByName = "data.mongodbatlas_project.test2"
+	dataSourcePluralName = "data.mongodbatlas_projects.test"
+)
 
-func TestAccProjectRSProject_basic(t *testing.T) {
+func TestAccProject_basic(t *testing.T) {
 	var (
-		group        admin.Group
-		orgID        = os.Getenv("MONGODB_ATLAS_ORG_ID")
-		clusterCount = "0"
-		projectName  = acc.RandomProjectName()
+		orgID          = os.Getenv("MONGODB_ATLAS_ORG_ID")
+		projectName    = acc.RandomProjectName()
+		projectOwnerID = os.Getenv("MONGODB_ATLAS_PROJECT_OWNER_ID")
 	)
+	commonChecks := map[string]string{
+		"name":          projectName,
+		"org_id":        orgID,
+		"cluster_count": "0",
+		"teams.#":       "2",
+	}
```

Comment:
could we add a check for `limits` attribute as well? seeing that it was tested in the data source tests

Reply:
sure, i'll do in the limit test